sale_service_fleet/report/project_report.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+from openerp import tools
+
+
+class ReportProjectTaskUser(models.Model):
+    _inherit = 'report.project.task.user'
+
+    vehicle_id = fields.Many2one(
+        comodel_name='fleet.vehicle', string='Vehicle', readonly=True)
+
+    def init(self, cr):
+        tools.sql.drop_view_if_exists(cr, 'report_project_task_user')
+        # Adds t.vehicle_id to select and group in tasks report view
+        cr.execute("""
```

Comment:
Please don't replace totally a method, or you kill the inheritance chain. Use a similar technique that I use in https://github.com/OCA/crm/pull/74/files#diff-66b637b1e994d4439cf089b62a5a0ed5R14